types/tx_test.go

```diff
@@ -0,0 +1,127 @@
+package types
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	cmn "github.com/tendermint/go-common"
+	ctest "github.com/tendermint/go-common/test"
+	wire "github.com/tendermint/go-wire"
+)
+
+func makeTxs(cnt, size int) Txs {
+	txs := make(Txs, cnt)
+	for i := 0; i < cnt; i++ {
+		txs[i] = cmn.RandBytes(size)
+	}
+	return txs
+}
+
+func randInt(low, high int) int {
+	off := cmn.RandInt() % (high - low)
```

Comment:
Good point, but then what is cmn.RandInt() for?  And actually quite a few functions that mirror stdlib?

I figured to use go-common for these until we reorganize everything, but am cool with any route also.  Maybe @ebuchman could clarify the proper way to do this.